src/workerd/server/tests/python/import_tests.bzl

```diff
@@ -23,7 +23,7 @@ const unitTests :Workerd.Config = (
           (name = "{}", pythonRequirement = ""),
         ],
         compatibilityDate = "2024-05-02",
-        compatibilityFlags = ["python_workers"],
+        compatibilityFlags = ["python_workers_development"],
```

Comment:
I wonder if we can do a find-and-replace on this string to generate the two variants of tests we talked about. I'll fiddle around with the bazel and make a PR.